main/src/ca/uwaterloo/flix/language/phase/Verifier.scala

```diff
@@ -491,7 +491,7 @@ object Verifier {
       val bodyType = visitExpr(exp2)(root, env + (sym -> letBoundType), lenv)
       checkEq(bodyType, tpe, loc)
 
-    case Expr.Stmt(exp1, exp2, tpe, _, loc) =>
+    case Expr.Stmt(_, exp2, tpe, _, loc) =>
       val secondType = visitExpr(exp2)
```

Comment:
wait who removed the call to exp1

Reply:
Looks like Magnus:

https://github.com/flix/flix/commit/5960f2ce77b2115de591c0951d513b6ac38e20c0#diff-a9f7c8437205f4387f8bd19b5f2cba2f397e1dc9c7046b5f286713bdcb66f8af

Reply:
🤔